targets/TARGET_Cypress/TARGET_PSOC6/psoc6csp/hal/src/cyhal_crc.c

```diff
@@ -36,19 +36,15 @@
 cy_rslt_t cyhal_crc_init(cyhal_crc_t *obj)
 {
     CY_ASSERT(NULL != obj);
-
-    memset(obj, 0, sizeof(cyhal_crc_t));
-    obj->resource.type = CYHAL_RSC_CRC;
-    return cyhal_crypto_reserve(&(obj->base), &(obj->resource));
+    return cyhal_crypto_reserve(&(obj->base), &(obj->resource), CYHAL_CRYPTO_CRC);
 }
 
 void cyhal_crc_free(cyhal_crc_t *obj)
 {
-    CY_ASSERT(NULL != obj || obj->resource.type != CYHAL_RSC_CRC);
+    CY_ASSERT(NULL != obj || obj->resource.type != CYHAL_RSC_CRYPTO);
```

Comment:
This function changed resource type from `CYHAL_RSC_CRC` to `CYHAL_RSC_CRYPTO`. `cyhal_crc_init()` changed to `CYHAL_CRYPTO_CRC`.

Should this be `CYHAL_CRYPTO_CRC` to match what's in `cyhal_crc_init()`?

Reply:
CYHAL_CRYPTO_CRC is a subtype of resource CYHAL_RSC_CRYPTO

Reply:
Thanks, makes sense.